contrib/storage-mongo/src/main/java/org/apache/drill/exec/store/mongo/MongoGroupScan.java

```diff
@@ -492,7 +492,7 @@ public ScanStats getScanStats() {
       MongoClient client = storagePlugin.getClient();
       MongoDatabase db = client.getDatabase(scanSpec.getDbName());
       MongoCollection<Document> collection = db.getCollection(scanSpec.getCollectionName());
-      long numDocs = collection.countDocuments();
+      long numDocs = collection.estimatedDocumentCount();
```

Comment:
If we will use `estimatedDocumentCount`, we should also replace `GroupScanProperty.EXACT_ROW_COUNT` for `ScanStats` creation below to avoid possible wrong results when we will rely on the returned row count.

Reply:
@vvysotskyi Can you create a quick PR with that since we already merged this?